README.md

```diff
@@ -15,3 +15,28 @@ It provides a function that takes two string arguments and returns a hash like t
 ````
 
 Please see [tests](./test/test.js) for more insights.
+
+
+### Example
+
+Here is how you can use this package in an _Angular_ project:
```

Comment:
It seems to me that there is nothing specific to Angular in this example. Am I missing something?

Reply:
How about:

## Install

```sh
npm install damerau-levenshtein
```

## Use with ES6 modules

```js
import * as levenshtien from 'damerau-levenshtein';

// ...

const lev = levenshtien('hello world', 'Hello World!');
// { steps: 4, relative: 0.3076923076923077, similarity: 0.6923076923076923 }
```

## Use with TypeScript

```ts
interface LevenshteinResponse {
  steps: number;
  relative: number;
  similarity: number;
}
```